doc/classes/FileAccess.xml

```diff
@@ -241,6 +257,14 @@
 				Returns a SHA-256 [String] representing the file at the given path or an empty [String] on failure.
 			</description>
 		</method>
+		<method name="get_unix_permissions" qualifiers="static">
+			<return type="int" />
+			<param index="0" name="file" type="String" />
+			<description>
+				Returns file UNIX permissions.
+				[b]Note:[/b] This method is implemented on iOS, Linux (X11) and macOS.
```

Comment:
It's not specific to X11 I guess, and should work on BSD?
```suggestion
				[b]Note:[/b] This method is implemented on iOS, Linux/BSD, and macOS.
```
Same for setter.

Reply:
Done.